packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -1967,28 +1967,30 @@ describe("<DateRangeInput>", () => {
             const defaultValue = [START_DATE, null] as DateRange;
 
             const { root, getDayElement } = wrap(<DateRangeInput defaultValue={defaultValue} onChange={onChange} />);
-            root.setState({ isOpen: true });
 
+            getStartInput(root).simulate("focus");
             getDayElement(START_DAY).simulate("click");
             assertInputTextsEqual(root, "", "");
             expect(onChange.called).to.be.true;
             expect(onChange.calledWith([null, null])).to.be.true;
         });
 
-        it(`Clearing only the start input (e.g.) invokes onChange with [null, <endDate>]`, () => {
+        it("Clearing only the start input (e.g.) invokes onChange with [null, <endDate>]", () => {
```

Comment:
"(e.g.)" throws me off here. not an example, that's actually what you're testing.